docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+```
+
+Finally, add `lbaas-...cern.ch` as an IP address in the LanDB set dashboard. 
+
+### Produce the certificates related to the landb-alias
+Read the [help page](https://ca.cern.ch/ca/Help/?kbid=021002) about Grid Host certs
+
+Then, request a cert using [dedicated page](https://ca.cern.ch/ca/host/HostSelection.aspx?template=EE2Host&instructions=auto). 
+In the current example, a cert for `compass-rucio.cern.ch` (corresponding to the LB `landdb-alias`) was requested:
+
+> [!tip] Certs pro move
+> Create a [grid Host certificate](https://ca.cern.ch/ca/host/Request.aspx?template=ee2host) linked to both servers and auth domains. Use the SAN box add as many domains as you want. Then you'll have one cert to rule them all:
+ 
+![[/img/grid-host-certs.png]]
+
+#### Create the `host`, `key`, `ca` and `GridCA` files
+
+As recommended in the [documentation](https://ca.cern.ch/ca/Help/?kbid=024100), execute:
+```sh
+openssl pkcs12 -in compass-rucio.p12 -clcerts -nokeys -out hostcert.pem
+
+openssl pkcs12 -in compass-rucio.p12 -nocerts -nodes -out hostkey.pem
+
+chmod 400 hostkey.pem 
+chmod 644 hostcert.pem 
+```
+These files have to be stored in the same path as described by the `RAW_SECRETS_SERVERS` variable used in the secrets creation.
+##### Important information about `ca.pem` and `GridCA.pem`
+An additional file, `ca.pem` (i.e. `CERN-bundle.pem`) must be created. 
+This file can be retrieved from lxplus:
+```sh
+cp /etc/pki/tls/certs/CERN-bundle.pem compass-rucio-certs/servers/ca.pem
+```
+The same goes for the `GridCA.pem` file: 
+```sh
+cp /etc/grid-security/certificates/CERN-GridCA.pem compass-rucio-certs/servers/GridCA.pem
+```
+
+### Add the certificates as secrets and mount them on the k8s pods
+Now it is possible to run something similar to the [servers script](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/scripts/rucio-servers-secret.sh?ref_type=heads) to add all the secrets to the cluster.
+
+The output should be of this form: 
+```sh
+Create and apply SERVER secrets
+sealedsecret.bitnami.com/rucio-servers-server-hostcert configured
+sealedsecret.bitnami.com/rucio-servers-server-hostkey configured
+sealedsecret.bitnami.com/rucio-servers-server-cafile configured
+.
+.
+.
+```
+The flux sync can be triggered by ***git pushing*** the content of the `SECRET` folder to the remote repository.
+## Rucio Auth
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers-auth.yaml?ref_type=heads).
+
+>The authentication component of Rucio is responsible for securely verifying the identities of users and services attempting to access the Rucio system. It ensures that only authorised entities can perform actions within the system.
+
+As for the servers, four steps are needed to deploy the authentication pod:
+1. Produce the Helm chart
+2. Setup the LB
+3. Produce the certificates related to the landb-alias used
```

Comment:
What is landb-alias here?

Reply:
Added more context for auth and clarified the steps needed to change the LB attributes.